reqs/tests/views_tests.py

```diff
@@ -210,3 +210,37 @@ def test_policies_counts_filter_by_multiple_topics(policy_setup):
     assert response['results'][1]['total_reqs'] == len(reqs[1])
     # reqs[1][1]
     assert response['results'][1]['relevant_reqs'] == 1
+
+
+@pytest.mark.django_db
+def test_excludes_nonpublic():
+    """The API endpoint should not include policies flagged as nonpublic"""
+    policies = [mommy.make(Policy, policy_number='0'),
+                mommy.make(Policy, policy_number='1', nonpublic=True)]
+    reqs = [mommy.make(Requirement, policy=policies[0], _quantity=3),
+            mommy.make(Requirement, policy=policies[1], _quantity=4)]
```

Comment:
Minor: naming these policies/requirements `public_policy` `nonpublic_policy`, etc. might make reading the logic a tad easier.